extensions/wikia/MercuryApi/MercuryApiHooks.class.php

```diff
@@ -87,20 +82,8 @@ public static function onArticleRollbackComplete( WikiPage $wikiPage, User $user
 	 * @return bool
 	 */
 	static public function onTitleGetSquidURLs( Title $title, Array &$urls ) {
-		global $wgServer;
-
 		if ( $title->inNamespaces( NS_MAIN ) ) {
-			// Mercury API call from Ember.js to Hapi.js e.g.
-			// http://elderscrolls.wikia.com/api/mercury/article/Morrowind
-			$urls[] =
-				$wgServer .
-				self::SERVICE_API_ROOT .
-				self::SERVICE_API_BASE .
-				self::SERVICE_API_ARTICLE .
-				$title->getPartialURL();
-
-			// Mercury API call from Hapi.js to MediaWiki e.g.
-			// http://elderscrolls.wikia.com/wikia.php?controller=MercuryApi&method=getArticle&title=Morrowind
+			// Request from browser to MediaWiki
 			// TODO: Remove one of these two below when it is decided if we do the switch to getPage() or drop it.
 			$urls[] = MercuryApiController::getUrl( 'getArticle', [ 'title' => $title->getPartialURL() ] );
```

Comment:
We would have to fix https://github.com/Wikia/api-tests/blob/master/apitests/Mercury/test_mercury_api.py#L13 and possibly some other places too. Let's not do that now.